Disable external downloads during configure. See Makefile for a comment on this.

--- third-party/CMakeLists.txt~	2024-11-25 13:24:09.000000000 +0100
+++ third-party/CMakeLists.txt	2025-04-02 14:28:04.676214149 +0200
@@ -5,7 +5,6 @@
 
 pop_security_flags() # remove security flags for third party, as we cannot guarantee their security enforcment
 
-include(CMake/external_json.cmake)
 # Add additional include directories to allow file to include rosbag headers
 include(${_rel_path}/realsense-file/config.cmake)
 
